### ISSUE TITLE
Feature request: Color schemes based on environment context

### ISSUE THREAD

**Issue by theodore-evans** [microsoft/vscode#118558]
I often develop with VS Code attached to a running container on a remote server (which is amazing!), but this often has me working in parallel in three different VS Code windows (or at least having them all open at once):

- Attached to the Docker container
- Running remotely via ssh
- Local

More than once I've started writing code in the wrong environment, and then had to copy it over, because all of my VS code windows look almost exactly the same (it also loves opening new windows for remote connections, but that's a separate issue)

It would be really helpful to be able to set up different themes for different remote environments (i.e. one theme for local, one for ssh remote, one for docker container), so that I could see at a glance which window I want to be working in, and swipe to the correct one without needing to check the remote status in the bottom left corner.
